debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleConnectorFilterIT.java

```diff
@@ -453,6 +477,140 @@ private void shouldApplySchemaAndTableExclusionsConfiguration(boolean useLegacyO
         }
     }
 
+    private void shouldApplyColumnInclusionsConfiguration(boolean useLegacyOption) throws Exception {
+        Field option = OracleConnectorConfig.COLUMN_INCLUDE_LIST;
+        if (useLegacyOption) {
```

Comment:
So if we never documented these legacy option names as per what your say, I'd suggest let's simply remove them altogether. Chances anyone used them are really low then, and given the connector is incubating, the remaining risk seems acceptable to me.

Reply:
Fixed, tests now only use the include/exclude option names for column filtering.